source/renderer/app/components/wallet/receive/WalletReceiveDialog.scss

```diff
@@ -165,3 +166,11 @@
     }
   }
 }
+
+.dialog {
+  :global {
+    .Dialog_title {
+      margin-bottom: 0;
+    }
+  }
+}
```

Comment:
@aleksandardjordjeviciohk please avoid adding global CSS here and instead define the following in Dialog.scss: 
```
.title + . subtitle { margin-top: -20px; }
```